src/scss/_mixins.scss

```diff
@@ -0,0 +1,35 @@
+/// Output all o-spacing features.
```

Comment:
All of the components that have a primary mixin set the primary mixin in the main.scss file, before calling it under `o-component-is-silent`. Could we move this over? 😬